test/unit/specs/users/cd-tickets.spec.js

```diff
@@ -0,0 +1,141 @@
+import vueUnitHelper from 'vue-unit-helper';
+import ticketList from '!!vue-loader?inject!@/users/cd-tickets';
+
+describe('Events/Tickets list component', () => {
+  let sandbox;
+  let TicketListWithMocks;
+  let MockEventsService;
+  let MockDojosService;
+  let MockUsersService;
+
+  beforeEach(() => {
+    sandbox = sinon.sandbox.create();
+    MockEventsService = {
+      loadEvents: sandbox.stub(),
+    };
+    MockDojosService = {
+      getUsersDojos: sandbox.stub(),
+    };
+    MockUsersService = {
+      getCurrentUser: sandbox.stub(),
+      getChildren: sandbox.stub(),
+    };
+    TicketListWithMocks = ticketList({
+      '@/events/service': MockEventsService,
+      '@/dojos/service': MockDojosService,
+      '@/users/service': MockUsersService,
+    });
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  describe('methods', () => {
+    describe('loadCurrentUser', () => {
+      it('should load the current user', (done) => {
```

Comment:
`async () => {` instead, then just use `await vm.loadCurrentUser();`, and remove `requireAnimationFrame` and `done()`.

Rinse, repeat. :P